pkg/streamingpromql/operators/functions/rate_increase.go

```diff
@@ -272,3 +278,74 @@ func rateSeriesValidator() RangeVectorSeriesValidationFunction {
 		lastCheckedMetricName = metricName
 	}
 }
+
+func delta(step *types.RangeVectorStepData, rangeSeconds float64, emitAnnotation types.EmitAnnotationFunc) (float64, bool, *histogram.FloatHistogram, error) {
+	fHead, fTail := step.Floats.UnsafePoints()
+	fCount := len(fHead) + len(fTail)
+
+	hHead, hTail := step.Histograms.UnsafePoints()
+	hCount := len(hHead) + len(hTail)
+
+	if fCount > 0 && hCount > 0 {
+		// We need either at least two histograms and no floats, or at least two floats and no histograms to calculate a delta.
+		// Otherwise, emit a warning and drop this sample.
+		emitAnnotation(annotations.NewMixedFloatsHistogramsWarning)
```

Comment:
Could you please add a test for this case? You should be able to extend or borrow from `TestRateIncreaseAnnotations` in `engine_test.go` for this.

Reply:
Added in b9f89fb9dd2f586bc2a148cb3a04da2afce430b5

Reply:
Note that sprintf removed in 350a0e100af8f0fa2d5e9a45cbda160973a5a897